### DISCUSSION TITLE
homekit (bridge) support

### DISCUSSION THREAD

**Discussion by martinschki** [unifi-utilities/unifi-common#201] [Q&A]
Hello!
First of, thank you for this great set of tools!
I installed pihole on my newly purchased UDMP over the weekend and it is just: working :-)

I do have a couple of questions though and I was wondering if anyone here can help me understand things a little better.

- I have installed pihole on its own VLAN. (VLAN only setting in UDM) according to the instructions. Why is this recommended? Why not just install it in an existing VLAN or on the main LAN? It is shown as "VLAN only"

- I need a DDNS service.
It seems that I don't get freeDNS to work...would anyone recommend Cloudflare?

- now to my main question:
I still haven't made up my mind regarding video surveillance but I'm quite intrigued by the doorbell thingy from ubnt. For me a main requirement is homekit though and ubnt doesn't support that natively.
How good is the homebridge integration working?? Is it running good on the UDM Pro? I would like to use the "Protect" features for analysis but for everyday it NEEDS to work with my families apple devices.

Thanks for opinions

**Comment by boostchicken** [comment]
You can put it on any vlan you like, I like all my podman containers on one vlan so they their own IP Block and it makes for sexy firwall ruoles
